### PR TITLE
Adding :BZ: 1923320 to doctring

### PR DESCRIPTION
**Pull request by swadeley**
Closed loop: BZ#1923320

Hello

just adding BZ number to docstring and UUID to BZ

[Bug 1923320](https://bugzilla.redhat.com/show_bug.cgi?id=1923320) - Global Registration: of FIPS RHEL8 host fails with Transaction error: no digest

### CODE REVIEW

**Comment by swadeley** [comment]
trigger: robottelo
pytest: tests/foreman/ui/test_host.py -k test_global_registration_upgrade_subscription_manager

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k test_global_registration_upgrade_subscription_manager

File: tests/foreman/ui/test_host.py [hunk 1]
```diff
@@ -1695,6 +1695,8 @@ def test_global_registration_upgrade_subscription_manager(
         4. Check subscription-manager was installed from repo_name
 
     :parametrized: yes
+
+    :BZ: 1923320
```
Comment (pondrejk) [inline tests/foreman/ui/test_host.py:1699]: also `customerscenario: true`

**Comment by devendra104** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k test_global_registration_upgrade_subscription_manager

File: tests/foreman/ui/test_host.py (hunk 1 above)
Comment (jyejare) [reply tests/foreman/ui/test_host.py:1699]: Yes the Customer Cases are attached to the BZ so its worth adding customerscenario testimony token.

**Review by jyejare — APPROVED** [review]
ACK with @pondrejk comments to address!

**Comment by swadeley** [comment]
> @swadeley I think the PRT failure is not related to the change in this PR, its just a docstring update.

The problem was introduced by me in https://github.com/SatelliteQE/robottelo/pull/9499
I did not properly handle the case of basesos and appstream repos being in a dictinary, and the need to extract just baseos while not breaking the rhel7 case where repo is in a string.
I have made another commit to address that. 
> 
> Please address @pondrejk comment and we are good to merge it.
Done that.

BTW, RHEL8 and RHEL9 are not yet supported by these fixtures:

module_os
https://github.com/SatelliteQE/robottelo/blob/ba336e6a79bfd89d2c956dc89ee71e0372b36de2/tests/foreman/ui/test_host.py#L147

os_path
https://github.com/SatelliteQE/robottelo/blob/ba336e6a79bfd89d2c956dc89ee71e0372b36de2/tests/foreman/ui/test_host.py#L157

might cases PRT to fail.

**Comment by swadeley** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k test_global_registration_upgrade_subscription_manager

**Comment by swadeley** [comment]
Prt build has passed! Merge me quick!

**Review by pondrejk — APPROVED** [review]

**Comment by pondrejk** [comment]
code quality tests now failing everywhere, possibly due to widgetastic update

**Comment by swadeley** [comment]
Thank you

**Pull request merged**